assets/queries/terraform/kubernetes/cluster_allows_unsafe_sysctls/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "a9174d31-d526-4ad9-ace4-ce7ddbf52e03",
+  "queryName": "Cluster Allows Unsafe Sysctls",
+  "severity": "HIGH",
+  "category": "Insecure Configurations",
+  "descriptionText": "A Kubernetes Cluster must not allow Unsafe Sysctls, to prevent a pod from having any influence on any other pod on the node, harming the node's health or gaining CPU or memory resources outside of the resource limits of a pod. This means the 'spec.securityContext.sysctls' must not have an Unsafe Sysctls and that the attribute 'allowedUnsafeSysctls' must be undefined.",
```

Comment:
```suggestion
  "descriptionText": "A Kubernetes Cluster must not allow unsafe sysctls, to prevent a pod from having any influence on any other pod on the node, harming the node's health or gaining CPU or memory resources outside of the resource limits of a pod. This means the 'spec.securityContext.sysctls' must not have an Unsafe Sysctls and that the attribute 'allowedUnsafeSysctls' must be undefined.",
```